app/components/video-block.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
```

Comment:
I admit there wasn't a readme with contributing instructions for enciv-home, so you may not have seen this. But please put a comment at the top with the link to the issue.

And, here are the new contributing and react guidelines -
[Contribute](https://github.com/EnCiv/.github/wiki/Contributing)
[React Coding and Style Guidlines](https://github.com/EnCiv/.github/wiki/React-Coding-and-Style-Guidelines)

Reply:
I Added comment at the top with the link to the issue , added ...otherP…
…rops in VideoBlock, changed defaultProps style, changed css based on Figma. Also I delete the package-lock file and test this component. But there is a thing left, I can't modify the ActionButton to make it looks like in Figma. I can't expand the button length. I tried add spaces on action-text left and right, but it did not work.